frontend/src/app/testResults/uploads/CsvUploadGuide.tsx

```diff
@@ -0,0 +1,271 @@
+                    <a
+                      href="https://SimpleReport.cdc.gov/assets/csv/SimpleReport-StandardCSV-ExampleData-20220509.csv"
+                      className="usa-link"
+                    >
+                      CSV template file
+                    </a>{" "}
+                    and{" "}
+                    <a href="/getting-started/testing-facilities/csv-schema">
+                      {" "}
+                      CSV schema documentation
+                    </a>
+                    .
+                  </p>
+                  <p>
+                    If you're starting from scratch, use the standard CSV as a
+                    template.
+                  </p>
+                </li>
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">
+                    Format your column headers to match the SimpleReport
+                    standard CSV
+                  </h4>
+                  <p>
+                    Whether you're modifying an existing file or creating a new
+                    one from scratch, be sure to include all column headers
+                    outlined in the standard file and documentation. Do not
+                    include any additional column headers. Make sure headers are
+                    written exactly as defined.
+                  </p>
+                  <p>
+                    Include column headers even if you don’t have data for every
+                    field.
+                  </p>
+                </li>
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">
+                    Enter values into your CSV file
+                  </h4>
+                  <p>
+                    Following the{" "}
+                    <a href="/getting-started/testing-facilities/csv-schema">
+                      {" "}
+                      CSV template guide
+                    </a>{" "}
+                   , enter properly formatted values in the relevant
+                    fields.
+                  </p>
+                </li>
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">Export your CSV</h4>
+                  <p>Export your properly formatted CSV with filled-in data.</p>
+                  <p>
+                    <em>
+                      Be sure to save your file as .CSV. SimpleReport CSV upload
+                      does not accept files saved as .XLS, .XLXS, or other
+                      formats.
+                    </em>
+                  </p>
+                </li>
+              </ol>
+            </section>
+            <section>
+              <h3
+                id="upload-a-csv"
+                className="font-body-lg border-top-1px border-ink margin-top-8 margin-bottom-6 padding-top-1"
+              >
+                Uploading a CSV file
```

Comment:
@lauraykerr what do you think about this section (image attached)?

<img width="509" alt="Screen Shot 2022-06-30 at 12 00 11 PM" src="https://user-images.githubusercontent.com/27730981/176724264-1cad2b9c-6a9a-4f11-9768-2ca2a1c4da15.png">

On ReportStream, these documents are located on the static site and are available to unauthenticated viewers. At least for now, though, anyone using the uploader in SimpleReport will already have logged in, navigated to the uploader, etc.

Reply:
@nathancrtr Good call out — I think we can scrap step #1. We also probably want to edit #2 (now #1) to say: 
**Navigate to "Upload spreadsheet**
You can find "Upload spreadsheet" under "Results" in the main navigation at the top of the page.